tests/ott_profile.py

```diff
@@ -0,0 +1,28 @@
+from oteltest import Telemetry
+from oteltest.telemetry import num_logs
+from ott_lib import project_path, trace_loop
+
+if __name__ == "__main__":
+    trace_loop(12)
+
+
+class ProfileOtelTest:
+    def environment_variables(self):
+        return {
+            "SPLUNK_PROFILER_ENABLED": "true",
+        }
+
+    def requirements(self):
+        return [project_path(), "oteltest"]
+
+    def wrapper_command(self):
+        return "opentelemetry-instrument"
+
+    def on_start(self):
+        pass
+
+    def on_stop(self, tel: Telemetry, stdout: str, stderr: str, returncode: int):
+        assert num_logs(tel)
```

Comment:
Some other tiny spot check that this smells like a profiler log?  Anything at all?

Reply:
Good idea. Done.